src/cape.cpp

```diff
@@ -165,7 +164,7 @@ void Cape::process(Eigen::MatrixXf& cloud_array, cv::Mat& seg_out,
     new_ps.fitPlane();
 
     // 4. Model fitting
-    if (new_ps.score > 100) {
+    if (new_ps.score > 1) {

```

Comment:
Is there reason why the previous value was high as 100  for Model fitting? Did it work before your changes?